src/modules/HappyToast.css

```diff
@@ -0,0 +1,38 @@
+.HappyToast {
+  -webkit-user-select: none; /* Safari */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* IE10+/Edge */
+  user-select: none; /* Standard */
+
+  position: absolute;
+  right: var(--component-margin);
+  bottom: calc(var(--tile-diam) * -2 - var(--component-margin));
+
+  border-radius: 4px;
+  box-shadow: var(--material-shadow);
+  background-color: #d5f6d8;
+  color: #28a745;
+  border-left-color: #28a745;
```

Comment:
I really need to stop putting raw colors in my css. Should really move all the toast colors to StylingSlice. These are really just notes to self as I go, in case you ever actually see this, btw.